themes/devfest-theme-hugo/assets/style/theme-2026.scss

```diff
@@ -0,0 +1,146 @@
+@use "root";
+@use "variables";
+
+@use "partials/base";
+@use "partials/header";
+@use "partials/main";
+@use "partials/footer";
+@use "partials/buttons";
+@use "partials/socials";
+@use "partials/jumbo";
+@use "partials/banner";
+@use "partials/hero";
+@use "partials/table_of_contents";
+@use "partials/speakers_shortcode";
+@use "partials/tickets";
+@use "partials/partners_group";
+@use "partials/tags";
+@use "partials/icons";
+@use "partials/type";
+@use "partials/form";
+@use "partials/album";
+@use "partials/paper";
+
+@use "pages/home";
+@use "pages/sessions";
+@use "pages/session";
+@use "pages/partners";
+@use "pages/partner";
+@use "pages/schedule";
+@use "pages/speakers";
+@use "pages/speaker";
+@use "pages/blogs";
+@use "pages/blog";
+@use "pages/kids";
+@use "pages/team";
+@use "pages/faq";
+@use "pages/code_of_conduct";
+@use "pages/with_banner";
+@use "pages/text_page";
+@use "pages/papers";
+
+:root {
+    --primary:#ac191c; //#083a7c//#44bb44 // #4775ff
+}  
+
+#registration  ~ table td,
```

Comment:
the lines from here on were added by the Chinese team of 2025, I'm not sure if they are strictly needed, as we did not have them in 2024, compare to the theme-2024.css . We should probably check what exactly they do and if we need them, we could incorporate them directly into the appropriate style files which are imported above.